db/seeds.rb

```diff
@@ -51,7 +51,7 @@
                               town: "Farnham",
                               county: "Not recorded",
                               postcode: "GU9 8UG",
-                              geolocation: "(51.2023732521965,0.814476304733643)")
+                              geolocation: "(51.2023732521965,-0.814476304733643)")
```

Comment:
this intentional? coordinates for farnham look like they should be +0.8ish

Reply:
[Wrong](https://www.google.com/search?ei=JDUqYJWBBMSS8gL8xZaYCg&q=51.2023732521965%2C0.814476304733643) vs [correct](https://www.google.com/search?q=51.2023732521965%2C-0.814476304733643)